CorsixTH/Lua/dialogs/edit_room.lua

```diff
@@ -1209,11 +1209,11 @@ function UIEditRoom:setDoorBlueprint(x, y, wall)
     flags = flags + 16 -- Use red palette rather than normal palette
   end
   if self.room_type.swing_doors then
-    for _, anim in ipairs(anim) do
-      anim:setAnimation(self.anims, 126, flags)
+    for _, anim2 in ipairs(anim) do
```

Comment:
This function is huge. Suggest breaking it up, which might avoid using silly variable names.